### DISCUSSION TITLE
Admin widgets disappear after installing Polylang

### DISCUSSION THREAD

**Discussion by rockyplanet** [holger1411/understrap#2114] [General]
Hello. I installed the Polylang plugin and now my widgets do not show up in the admin area, but they show content on the website. I tested a default wordpress theme and Understrap parent theme v1.2.2 they show up, but when I activate the understrap premium child theme blog v1.1.0, they disappear. There are no errors. Does anyone in here have an idea on how to fix this?

**Comment by bacoords** [comment]
Can you explain what you mean when you say the widgets don't show up? The screen doesn't appear in the menu, or it does but the page loads blank? Basically any screenshots would be helpful, as well as any output in the javascript console and/or the debug.log file.

**Reply by rockyplanet** [reply]
I've narrowed down the problem. I found that there seems to be some sort of incompatibility between polylang and the litespeed cache plugin and it most likely has nothing to do with the understrap blog theme. I'll work it out with the litespeed cache team to see if they can resolve it.

**Comment by rockyplanet** [comment]
This problem was resolved with the help of litespeed cache support team. I reset the plugin to default settings and then went from there.
